js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,14 @@
+
+function lastIndexOf(...args) {
+  const item = args[0];
+  const list = [...args[1]];
+  const keys = [];
+  list.forEach((element, index) => {
+    if (element === item) keys.push(index);
```

Comment:
You can directly return the index. I think there is no need to push the same in an array and then find the maximum.
After forEach you can simply return -1.